packages/frontend/src/pages/Relay/Relay.tsx

```diff
@@ -0,0 +1,213 @@
+
+const useStyles = makeStyles((theme: any) => ({
+  root: {
+    maxWidth: '680px',
+    margin: '0 auto',
+  },
+  header: {
+    marginBottom: '4rem',
+    textAlign: 'center',
+  },
+  form: {
+    display: 'block',
+    marginBottom: '4rem',
+  },
+  card: {
+    marginBottom: '4rem',
+  },
+  loader: {
+    marginTop: '2rem',
+    textAlign: 'center',
+  },
+  notice: {
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column',
+  },
+}))
+
+export const Relay: FC = () => {
+  const styles = useStyles()
+  const { sdk, networks, txConfirm } = useApp()
+  const { checkConnectedNetworkId } = useWeb3Context()
+  const { queryParams } = useQueryParams()
+  const [txHash, setTxHash] = useState<string>(() => {
+    return queryParams?.txHash as string || ''
+  })
+  const [loading, setLoading] = useState<boolean>(false)
+  const [error, setError] = useState<string>('')
+  const [success, setSuccess] = useState<string>('')
+  const [selectedNetwork, setSelectedNetwork] = useState<Network>(l2Networks[0])
+  // const { selectedNetwork, selectBothNetworks } = useSelectedNetwork()
+
+  useEffect(() => {
+    try {
+      updateQueryParams({
+        txHash: txHash || ''
+      })
+    } catch (err: any) {
+      console.error(err)
+    }
+  }, [txHash])
+
+  async function handleSubmit(event: ChangeEvent<any>) {
+    event.preventDefault()
+    try {
+      setLoading(true)
+      setError('')
+      setSuccess('')
+      await new Promise((resolve, reject) => {
+        const run = async () => {
+          console.log('selectedNetwork', selectedNetwork)
+          const isNetworkConnected = await checkConnectedNetworkId(l1Network.networkId)
+          if (!isNetworkConnected) {
+            throw new Error('wrong network connected')
+          }
+          const l1Wallet = await sdk.getSignerOrProvider(l1Network.slug)
+          const l2Wallet = await sdk.getSignerOrProvider(selectedNetwork.slug)
+          console.log('reactAppNetwork', reactAppNetwork)
+          console.log('l1Wallet', l1Wallet)
+          console.log('l2Wallet', l2Wallet)
+          // example commit txs
+          // let commitTxHash = '0x86f1dfc3ced80aa27c1116ebda552b5cd5009eac5f6f3c2d01a521faae398677' // base ETH
+          // let commitTxHash = '0xb3b1e32b65aab3dd5374130fe92ff3108fef1d5d61be62bb99e041e596b64265' // optimism ETH
+          // let commitTxHash = '0x5a75ff2131895ec42da3bc851df8854683f20bb7dadb6f149e527da021d91456' // polygon ETH
+          // let commitTxHash = '0x43c5f6baf4100fbd7924a9f0664f20ed8b36bf3feb4218aea41d1e01f5076bf4' // arbitrum ETH
+          // let commitTxHash = '0x886158d6ce41723af5364b9f89d51416c09345ff920c8e7af23dfd67ccd34da2' // nova ETH
+          // let commitTxHash = '0xc02c7337289afbc5d73199e26585da58b7cf77334f8d49255751a6da421a13a4' // gnosis ETH
+          let commitTxHash = ''
+          if (!commitTxHash) {
+            const event = await getTransferCommittedEventForTransferId(selectedNetwork.slug, 'ETH', txHash)
```

Comment:
I think 'ETH' should be a passed-in token? Unless it is ignored within the called method. If so, maybe just note that somewhere.